docs/sidebars.js

```diff
@@ -54,7 +54,8 @@ module.exports = {
         { type: "doc", id: "reference/adapter/typeorm/index" },
         { type: "doc", id: "reference/adapter/mongodb/index" },
         { type: "doc", id: "reference/adapter/firebase/index" },
-        { type: "doc", id: "reference/adapter/dgraph/index" },
+        { type: "doc", id: "reference/adapter/dynamodb/index" },
```

Comment:
I forgot to link to the new **DynamoDB** docs in [my last PR](https://github.com/nextauthjs/next-auth/pulls?q=is%3Apr+is%3Aclosed) 😅